src/components/RemoveCars.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { deleteCarAction } from '../store/mainpage_reducer';
+import './RemoveCars.css';
+
+const Remove = () => {
```

Comment:
- [ ] For some reason, I can't remove a car:

![image](https://user-images.githubusercontent.com/57421823/200626780-639ea1b7-98d4-4d09-bc49-e5356f22f4c8.png)

![image](https://user-images.githubusercontent.com/57421823/200626835-cba0f6bd-7b53-4989-a949-4dbd95c24e8c.png)

![image](https://user-images.githubusercontent.com/57421823/200626895-88918342-3398-4dbe-a8ca-0ba24dc0a96e.png)


Kindly fix this bug, so your app works as intended 💪